util/appveyor/build.cmd

```diff
@@ -0,0 +1,21 @@
+@echo off
+:: To build extensions for 64 bit Python 3, we need to configure environment
+:: variables to use the MSVC 2010 C++ compilers from GRMSDKX_EN_DVD.iso of:
+:: MS Windows SDK for Windows 7 and .NET Framework 4
+::
+:: More details at:
+:: https://github.com/cython/cython/wiki/64BitCythonExtensionsOnWindows
```

Comment:
Those instructions look outdated, in particular they are missing that the MSVC version to build Python 3.5 changed. The [link above the table](https://blog.ionelmc.ro/2014/12/21/compiling-python-extensions-on-windows/) does use MSVC Express instead of the ``GRMSDKK..`` thing. So I'm not sure that the simplifications below are OK.